src/types/tracker.d.ts

```diff
@@ -83,15 +83,7 @@ declare class Tracker {
       sortOrder?: string;
       sortBy?: string;
       section?: string;
-    },
-    networkParameters?: NetworkParameters
-  ): true | Error;
-
-  trackSearchResultsLoaded(
```

Comment:
GH doesn't let me add a comment on line 74. Should that be changed to trackSearchResultsLoaded from trackSearchResultsLoadedV2?